docs/getting_started.md

```diff
@@ -0,0 +1,140 @@
+* A `Decidim::Organization` named `Decidim Staging`. You probably want to change its name and hostname to match your needs.
+* A `Decidim::User` acting as an admin for the organization, with email `admin@example.org` and password `decidim123456`.
+* A `Decidim::User` that also belongs to the organization but it's a regular user, with email `user@example.org` and password `decidim123456`.
+
+This data won't be created in production environments, if you still want to do it, run:
+
+```
+$ SEED=true rails db:setup
+```
+
+You can now start your server!
+
+```
+$ bin/rails s
+```
+
+Visit [http://localhost:3000](http://localhost:3000) to see your app running.
+
+## Configuration & setup
+
+Decidim comes pre-configured with some safe defaults, but can be changed through the `config/initializers/decidim.rb` file in your app. Check the comments there or read the comments in [the source file](https://github.com/AjuntamentdeBarcelona/decidim/blob/master/decidim-core/lib/decidim/core.rb) (the part with the `config_accessor` calls) for more up-to-date info.
+
+We also have other guides on how to configure some extra features: 
+
+- [Social providers integration](https://github.com/AjuntamentdeBarcelona/decidim/blob/master/docs/social_providers.md): Enable sign up from social networks.
+- [Analytics](https://github.com/AjuntamentdeBarcelona/decidim/blob/master/docs/analytics.md): How to enable analytics
+
+## Creating an Authorization handler
+
+The most common use case is validating a user against a census, so that you can differentiate users living in your city from users living outside.
+
+The generator will have created an `ExampleAuthorizationHandler` so you can learn how to implement your own. You'll usually want to define some fields that you will use to authenticate against a census (for example, an ID and a Postal Code). In this class you'll need to write the logic to validate the user against the census. See the documentation for [the parent class](https://github.com/AjuntamentdeBarcelona/decidim/blob/master/decidim-core/app/services/decidim/authorization_handler.rb) and a [live example in Decidim Barcelona](https://github.com/AjuntamentdeBarcelona/decidim-barcelona/blob/master/app/services/census_authorization_handler.rb).
+
+You'll need to reference it from the Decidim initializer:
+
+```ruby
+# config/initializers/decidim.rb
+
+config.authorization_handlers = [<my authorization handler class>]
+```
+
+## Deploy
+
+Once you've generated the Decidim app you might need to do some changes in order to deploy it to Heroku. You can check [`codegram/decidim-deploy-heroku`](https://github.com/codegram/decidim-deploy-heroku) for an opinionated example of things to do before deploying to Heroku.
+
+Once you've successfully deployed your app to your favorite platform, you'll need to create your `System` user. First, access your production Rails console. In Heroku you'd need to run:
+
+```
+heroku run rails console --app <my heroku app name>
```

Comment:
Generalize this so it isn't tied to Heroku (just say you need to access Rails console in your production environment)